### PR TITLE
Import nn in jax/__init__.py.

### PR DESCRIPTION
**Pull request by tomhennigan**
Not sure when this changed, but last week `jax.nn.softmax` (et al) would work.

### CODE REVIEW

**Comment by mattjj** [comment]
Thanks, Tom!
